dsl/pace/dsl/stencil.py

```diff
@@ -263,6 +310,10 @@ def __call__(
         )
 
 
+def _stencil_object_name(stencil_object: gt4py.StencilObject) -> str:
+    return type(stencil_object).__name__
```

Comment:
Is this going to give unique names for each stencil? If it is, can you add a `#` comment to that effect, e.g. saying the dynamically-generated gt4py stencil class names include a hash of the compile-time constants?